x-pack/plugins/apm/scripts/test/api.js

```diff
@@ -22,6 +22,11 @@ const { argv } = yargs(process.argv.slice(2))
     type: 'boolean',
     description: 'Run tests with trial license',
   })
+  .option('rules', {
```

Comment:
why do we need a rules "license"/config? is it really needed to create an action for the rule to run/update? that would be unexpected to me.

Reply:
yeah, we don't need it, I was doing some experiments and forgot to revert it.